ckan_cloud_operator/providers/storage/manager.py

```diff
@@ -1,9 +1,46 @@
+from ckan_cloud_operator.config import manager as config_manager
 from ckan_cloud_operator.providers import manager as providers_manager
+
 from .minio.constants import PROVIDER_ID as minio_provider_id
-from .constants import PROVIDER_SUBMODULE
+from .s3.constants import PROVIDER_ID as s3_provider_id
+from .constants import PROVIDER_SUBMODULE, CONFIG_NAME
 
 
 def initialize(interactive=False, provider_id=None, storage_suffix=None, use_existing_disk_name=None, dry_run=False):
+    from ckan_cloud_operator.providers.cluster import manager as cluster_manager
+    from ckan_cloud_operator.providers.cluster.gcloud.constants import PROVIDER_ID as gcloud_provider_id
+    from ckan_cloud_operator.providers.cluster.aws.constants import PROVIDER_ID as aws_provider_id
+
+    if not provider_id and interactive:
+        cluster_provider_id = cluster_manager.get_provider_id()
+
+        config_manager.interactive_set(
+            {'use-cloud-native-storage': True},
+            secret_name=CONFIG_NAME,
+            interactive=interactive
+        )
+        if config_manager.get('use-cloud-native-storage', secret_name=CONFIG_NAME):
```

Comment:
Why not move these following initializations to the matching {s3/gcp}/manager.py initialize functions?